.ado/templates/react-native-init.yml

```diff
@@ -98,13 +98,13 @@ steps:
 
   - ${{ if eq(parameters.useNuget, true) }}:
     - script: |
-        npx --yes react-native-windows-init@latest --verbose --version $(npmVersion) --overwrite --language ${{ parameters.language }} --projectType ${{ parameters.projectType }} ${{ parameters.additionalInitArguments }} --experimentalNuGetDependency true --nuGetTestFeed $(System.DefaultWorkingDirectory)\NuGetTestFeed
```

Comment:
I think the intention here was that the new CLI bits would be published to Verdaccio. Looking above, I think the issue might be that we are setting the yarn config to use Verdaccio as the registry, but not doing the same for npm, which npx will use.

Reply:
Right, trying to fix that now, but npx is being stubborn.